pypi/data-processing/src/data_store_process/minio_store_process.py

```diff
@@ -170,6 +170,18 @@ def text_manipulate(
                     task_id=id,
                     create_user=req_json["creator"],
                 )
+            elif file_extension == "web":
+                # 处理.web文件
+                result = await web_handle.text_manipulate(
```

Comment:
@wangxinbiao as discussed,the `.web` file have the `web crawler configurationas`. Take this as as an example: 


```json
{"url":"http://1.1.1.1","interval_time":500,"resource_types":["type1","type2"],"max_depth":5,"max_count":10,"exclude_sub_urls":["url1","url2"],"include_sub_urls":["url3","url4"],"exclude_img_info":{"weight":100,"height":200}}
```

Reply:
ok